packages/components/.storybook/main.js

```diff
@@ -9,7 +9,7 @@ module.exports = {
         getAbsolutePath('@storybook/addon-viewport'),
         getAbsolutePath('@storybook/addon-actions'),
     ],
-    staticDirs: ['../public'],
+    staticDirs: ['../public', { from: '../../suite-data/files', to: '/static' }],
```

Comment:
This goes outside of the package. Is it ok? Maybe we can ask @Nodonisko if this is ok, or how to do it correctly. :pray:

Reply:
Since this is basically static path in bundler settings and not direct dependency I think it's fine